AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAssertImagePreviewActionGroup.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAssertImagePreviewActionGroup">
+        <click selector="{{AdobeStockSection.imagesThumbnails}}" stepKey="clickOnThumbnail"/>
+        <grabAttributeFrom selector="{{AdobeStockSection.imagePreview}}" userInput="src" stepKey="getUrlImageFromCmsPage"/>
+        <click selector="{{AdobeStockSection.savePreview}}" stepKey="clickOnSavePreview"/>
+        <waitForPageLoad stepKey="waitForPageLoad" />
+        <grabAttributeFrom selector="{{AdobeStockSection.selectedImagePreview}}" userInput="alt" stepKey="savedImage" />
+        <assertContains stepKey="AssertImageInMediaGallery">
+            <expectedResult type="variable">$savedImage</expectedResult>
+            <actualResult type="variable">$getUrlImageFromCmsPage</actualResult>
+        </assertContains>
+    </actionGroup>
+</actionGroups>
```

Comment:
I believe here and in the next action groups should be new lines at the end of files

Reply:
ok